include/util/geojson_debug_logger.hpp

```diff
@@ -105,6 +106,8 @@ class GeojsonLogger
         std::lock_guard<std::mutex> guard(lock);
         ofs.open(logfile, std::ios::binary);
 
+        ofs << std::setprecision(12);
```

Comment:
Well, since the only purpose of this logger is to output map geometry, and the guard manages the stream and does the writing, I felt like a general increase in accuracy would not hurt

Reply:
There's a state restorer for this in Boost

http://www.boost.org/doc/libs/1_64_0/libs/io/doc/ios_state.html